src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -294,35 +273,35 @@
     text: >
         az deployment create --location WestUS --template-uri https://myresource/azuredeploy.json --parameters @myparameters.json
   - name: Create a deployment at subscription scope from a local template file, using parameters from a JSON string.
-    text: |
-        az deployment create --location WestUS --template-file azuredeploy.json  \\
+    text: >
+        az deployment create --location WestUS --template-file azuredeploy.json \\
             --parameters "{ \\"policyName\\": { \\"value\\": \\"policy2\\" }}"
   - name: Create a deployment at subscription scope from a local template, using a parameter file, a remote parameter file, and selectively overriding key/value pairs.
     text: >
-        az deployment create --location WestUS --template-file azuredeploy.json  \\
+        az deployment create --location WestUS --template-file azuredeploy.json \\
             --parameters @params.json --parameters https://mysite/params.json --parameters MyValue=This MyArray=@array.json
   - name: Create a deployment at subscription scope from a template-spec
     text: >
-        az deployment create --location WestUS --template-spec "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myRG/providers/Microsoft.Resources/templateSpecs/myTemplateSpec/versions/1.0"
+        az deployment create --location WestUS --template-spec "/subscriptions/MySubscriptionID/resourceGroups/myRG/providers/Microsoft.Resources/templateSpecs/myTemplateSpec/versions/1.0"
```

Comment:
I'm not sure whether the placeholder `MySubscriptionID` is clearer than the sample value `00000000-0000-0000-0000-000000000000`. The sample value may make it easier for customers to understand the format of the subscription ID